site/content/en/docs/drivers/includes/vmware_windows_usage.inc

```diff
@@ -0,0 +1,13 @@
+## Requirements
+
+*VMware Workstation
+
+## Driver Installation
+
+First install the [VMware Workstation](https://www.vmware.com/in/products/workstation-pro/workstation-pro-evaluation.html) 
+
+After installation, start a cluster using the vmware driver:
+
+```shell
+minikube start --memory=4096 --cpus=4 --vm-driver vmware
```

Comment:
no need for 4GB and 4 Cpus options here.. minikube should work with Default Auto-select values. (it is better not to set it so minikube sets them for u automatically)

Reply:
sure I'll edit that. Thanks!